# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../audacity/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- audacity-src-1.2.4/lib-src/soundtouch/include/SoundTouch.h.vanilla	2006-03-04 17:02:06.681246500 +0100
+++ audacity-src-1.2.4/lib-src/soundtouch/include/SoundTouch.h	2006-03-04 17:02:27.482546500 +0100
@@ -146,7 +146,7 @@
     static const char *getVersionString();
 
     /// Get SoundTouch library version Id
-    static uint SoundTouch::getVersionId();
+    static uint getVersionId();
 
     /// Sets new rate control value. Normal rate = 1.0, smaller values
     /// represent slower rate, larger faster rates.
